src/components/panel/panel.js

```diff
@@ -895,7 +898,12 @@ MdPanelService.prototype.create = function(config) {
   this._config.scope.$on('$destroy', angular.bind(panelRef, panelRef.detach));
 
   if (this._config.groupName) {
-    panelRef.addToGroup(this._config.groupName);
+    if (angular.isString(this._config.groupName)) {
+      this._config.groupName = [this._config.groupName];
```

Comment:
It seems like the `_groupName` got converted into an array. I think it's more appropriate to call it something like `_groups` since an array implies multiple groups.